test/models/user_test.rb

```diff
@@ -320,6 +320,32 @@ def setup_user(operation)
     assert_includes record.errors.keys, :admin
   end
 
+  test "log of password change should be saved" do
+    user = FactoryGirl.create(:user)
+    user.password = nil
```

Comment:
This is rather hacky, reload the object from the database instead (`User.find_by_id(user.id)`) and add a comment to state why (i.e. to clear the value of user.password).